docs/types.md

```diff
@@ -72,10 +72,11 @@ Underlying Type: String
 
 An email address.
 
-#### Course Email Specification (CourseEmailSpec)
+#### Course User Reference (CourseUserReference)
 
 When used in the context of a course,
-a `CourseEmailSpec` can be used to generalize email recipients.
+a `CourseUserReference` can be used to generalize targeting users
+(e.g, email recipients).
```

Comment:
Because of this generalization, I would move this section up a level and put it above email (and then you can reference it in email).